maven-plugins/assembly-extension/README.md

```diff
@@ -69,6 +74,9 @@ and `helidon` as configured `containerDescriptorHandler`.
         <containerDescriptorHandler>
             <handlerName>helidon</handlerName>
         </containerDescriptorHandler>
+        <containerDescriptorHandler>
+            <handlerName>metaInf-services</handlerName>
```

Comment:
Maybe add a comment to state that this is built-in the maven-assembly plugin.

We could potentially add the link to the Maven [documentation](https://maven.apache.org/plugins/maven-assembly-plugin/examples/single/using-container-descriptor-handlers.html).

The comment can also state that Helidon uses META-INF/services and thus this is effectively required.